src/main/java/gregtech/api/worldgen/populator/SurfaceRockPopulator.java

```diff
@@ -38,10 +37,7 @@ public SurfaceRockPopulator(Material material) {
 
     @Override
     public void loadFromConfig(JsonObject object) {
-        DustMaterial material = OreConfigUtils.getMaterialByName(object.get("material").getAsString());
-        if (!(material instanceof DustMaterial))
-            throw new IllegalArgumentException("Only dust materials are supported for surface rocks");
-        this.material = material;
+        this.material = OreConfigUtils.getMaterialByName(object.get("material").getAsString());
```

Comment:
Should we sanity-check object.get() and/or getAsString() prior to passing into getMaterialByName?

Reply:
Fair, thanks.